docs/providers/aws/events/cognito-user-pool-trigger.md

```diff
@@ -0,0 +1,139 @@
+<!--
+title: Serverless Framework - AWS Lambda Events - CloudWatch Log
+menuText: Cognito User Pool Trigger
+menuOrder: 10
+description:  Setting up AWS Cognito User Pool Triggers with AWS Lambda via the Serverless Framework
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/aws/events/)
```

Comment:
😄 Yes, for some reason it's actually not auto-generated. Don't know why though 😄